pynndescent/rp_trees.py

```diff
@@ -708,7 +753,7 @@ def make_forest(data, n_neighbors, n_trees, rng_state, angular=False):
             flatten_tree(make_tree(data, rng_state, leaf_size, angular), leaf_size)
             for i in range(n_trees)
         ]
-    except (RuntimeError, RecursionError, SystemError):
```

Comment:
Probably better to allow the wider catch here I presume. And hopefully the tree recursion issues have been largely resolved at this point, so hopefully this will largely not be needed.